scripts/build-libdwarf.sh

```diff
@@ -9,11 +9,14 @@ fi
 
 prefix=$1
 
```

Comment:
Does conda have pre-built libdwarf and libelf we could use instead now?

Reply:
They have elfutils and I started creating a libdwarf recipe.  I had a question for you.  Latest libdwarf has an option to be built without libelf.  Are you fully using both or could we transition to using only latest libdwarf?

Reply:
Also, if we can use latest libdwarf, can we bump to using that along with switching or would you want me to build the current version first?  It's not a huge deal to build the old one first but if we can bump to the latest, its a little less work.

Reply:
https://github.com/davea42/libdwarf-code/releases/tag/libdwarf-0.1.1 is the version where libdwarf stopped needing libelf.  Unfortunately the github tag comments aren't as insightful as what he posts in his website at https://www.prevanders.net/dwarf.html#releases

Reply:
I looked at the `tracerv` code again and it looks like we use both.

I think it's probably fine to just bump it to latest? @a0u any input?

Reply:
Bumping libdwarf should be fine.  The current version was just the most recent when we started development.

And to confirm, we do use both libelf and libdwarf.  The stack reconstructor falls back to the ELF symbol table if DWARF debugging info is unavailable.

Reply:
Ok.  I'll finish submitting a PR to conda-forge to add libdwarf latest and we can do the bump in a follow-on.

Reply:
https://github.com/conda-forge/staged-recipes/pull/18638 is the PR to add libdwarf to conda-forge.  Once it has merged, I'll come back through and start a PR that bumps them and switches over to using the conda packages.